test/evergreen.yml

```diff
@@ -2326,8 +2326,8 @@ tasks:
             done
 
   - name: csuite-long-running
-    # Set 5 hours timeout (60 * 60 * 5)
-    exec_timeout_secs: 18000
+    # Set 6 hours timeout (60 * 60 * 6 = 21600)
```

Comment:
To better understand the situation, could you elaborate on the reasons for increasing the timeout?

Reply:
"wt load" uses checkpoint with the target option. That had to be changed to use checkpoint without the target option which is less efficient.

See https://spruce.mongodb.com/task/wiredtiger_ubuntu2004_msan_csuite_long_running_patch_6361ae9f7afd8f2d22ea7c8267206010bcea4044_66a92af320551a00077db7f0_24_07_30_18_03_32/logs?execution=0 in my patch 223 Test WT-12492 on all variants. With the target option csuite-long-running ! Ubuntu 20.04 MSAN took 4 hours and 31 minutes (base commit duration). Without it the task timed out.

When I increased the timeout from 5 hours to 6 hours it passed. See https://spruce.mongodb.com/task/wiredtiger_ubuntu2004_msan_csuite_long_running_patch_6361ae9f7afd8f2d22ea7c8267206010bcea4044_66a98897c18e0d0007128a01_24_07_31_00_43_05/logs?execution=0 csuite-long-running ! Ubuntu 20.04 MSAN in my patch 225 Test WT-12492  csuite-long-running.

This passing run took less than 5 hours. That shows the test time varies. Sometimes csuite-long-running ! Ubuntu 20.04 MSAN passes in 5 hours, sometimes it times out. It needs to pass all the time so I kept the timeout change.

Reply:
Thanks, @tod-johnson-mongodb for the detailed explanation.